src/components/Card.js

```diff
@@ -54,7 +55,10 @@ export const Card = () => {
 
     const ssid = escape(network.ssid);
     const password = escape(network.password);
-    setQrvalue(`WIFI:T:${network.encryptionMode};S:${ssid};P:${password};;`);
+    const hiddenSSID = network.hiddenSSID;
```

Comment:
No need to create this constant, just use `network.hiddenSSID` directly in the string template below.

Reply:
Done, thanks!